src/views/privacypolicy-apps/privacypolicy-apps.jsx

```diff
@@ -0,0 +1,313 @@
+                        <FormattedMessage id="privacyApps.legalSharingDescription" />
+                    </dd>
+                </dl>
+            </section>
+            <section id="community">
+                <dl>
+                    <span className="nav-spacer" />
+                    <h3>
+                        <FormattedMessage id="privacyApps.communityHeader" />
+                    </h3>
+                    <p>
+                        <FormattedMessage
+                            id="privacyApps.communityParagraph"
+                            values={{
+                                privacyPolicyLink: (
+                                    <a href="/privacy_policy/">
+                                        <FormattedMessage id="privacyApps.privacyPolicyLinkText" />
+                                    </a>
+                                )
+                            }}
+                        />
+                    </p>
+                </dl>
+            </section>
+            <section id="students">
+                <dl>
+                    <span className="nav-spacer" />
+                    <h3>
+                        <FormattedMessage id="privacyApps.studentsHeader" />
+                    </h3>
+                    <p>
+                        <FormattedMessage id="privacyApps.coppa" />
+                    </p>
+                    <p>
+                        <FormattedMessage id="privacyApps.ferpa" />
+                    </p>
+                </dl>
+            </section>
+            <section id="eea">
+                <dl>
+                    <span className="nav-spacer" />
+                    <h3>
+                        <FormattedMessage id="privacyApps.eeaHeader" />
+                    </h3>
+                    <p>
+                        <FormattedMessage id="privacyApps.eeaIntro" />
+                    </p>
+                    <dt>
+                        <FormattedMessage id="privacyApps.accessTitle" />
+                    </dt>
+                    <dd>
+                        <FormattedMessage id="privacyApps.accessDescription" />
+                    </dd>
+                    <dt>
+                        <FormattedMessage id="privacyApps.objectionTitle" />
+                    </dt>
+                    <dd>
+                        <FormattedMessage id="privacyApps.objectionDescription" />
+                    </dd>
+                    <dt>
+                        <FormattedMessage id="privacyApps.deletionTitle" />
+                    </dt>
+                    <dd>
+                        <FormattedMessage
+                            id="privacyApps.deletionDescription"
+                            values={{
+                                helpEmail: (
+                                    <a href="mailto:help@scratch.mit.edu">
+                                        help@scratch.mit.edu
+                                    </a>
+                                )
```

Comment:
Since this value is used in 3 places, and theoretically could change someday, I think it would make sense to put a constant at the top of the file to store either the literal text `help@scratch.mit.edu` or the whole `<a href=...` element.